src/addons/dragAndDrop/styles.less

```diff
@@ -2,7 +2,6 @@
 
 .rbc-addons-dnd {
   .rbc-row-content {
-    pointer-events: none;
```

Comment:
I agree that `pointer-events: none` is a bit problematic here, but removing this introduces new problems when doing DnD. Could it be added only when dragging is active?

Reply:
can you specify which problems occur? I didn't notice any so far.

Reply:
Part of the day slot in the month view becomes undroppable:

![demo](https://user-images.githubusercontent.com/3599069/34936652-86221bb2-f9e2-11e7-81a8-0e27daa5447f.gif)

Reply:
ahh, I see, I'll look into this shortly with the latest calendar update.